clean/sulphur_pd_cprr.py

```diff
@@ -0,0 +1,51 @@
+import sys
+
+sys.path.append("../")
```

Comment:
You can remove `sys.path.append('../')` for all scripts now that we have Deba.

Reply:
Thanks